tests/mocks/defaultMockData.json

```diff
@@ -0,0 +1,4701 @@
+{
+    "/config.json": {
+        "headers": [
+            {
+                "name": "accept-ranges",
+                "value": "bytes"
+            },
+            {
+                "name": "cache-control",
+                "value": "public, max-age=0"
+            },
+            {
+                "name": "content-length",
+                "value": "163"
+            },
+            {
+                "name": "content-type",
+                "value": "application/json; charset=UTF-8"
+            },
+            {
+                "name": "date",
+                "value": "Fri, 04 Feb 2022 14:29:00 GMT"
+            },
+            {
+                "name": "etag",
+                "value": "W/\"a3-17ec5219318\""
+            },
+            {
+                "name": "last-modified",
+                "value": "Fri, 04 Feb 2022 14:26:58 GMT"
+            },
+            {
+                "name": "x-powered-by",
+                "value": "Express"
+            }
+        ],
+        "statusCode": 200,
+        "statusText": "OK",
+        "body": {
+            "environment": "runtime",
+            "managementApiUrl": "https://contoso.management.azure-api.net",
+            "backendUrl": "https://contoso.developer.azure-api.net"
+        }
+    },
+    "/config-apim.json": {
+        "headers": [
+            {
+                "name": "accept-ranges",
+                "value": "bytes"
+            },
+            {
+                "name": "cache-control",
+                "value": "public, max-age=0"
+            },
+            {
+                "name": "content-length",
+                "value": "99"
+            },
+            {
+                "name": "content-type",
+                "value": "application/json; charset=UTF-8"
+            },
+            {
+                "name": "date",
+                "value": "Fri, 04 Feb 2022 14:29:00 GMT"
+            },
+            {
+                "name": "etag",
+                "value": "W/\"63-17ea11e2f9f\""
+            },
+            {
+                "name": "last-modified",
+                "value": "Fri, 28 Jan 2022 14:36:57 GMT"
+            },
+            {
+                "name": "x-powered-by",
+                "value": "Express"
+            }
+        ],
+        "statusCode": 200,
+        "statusText": "OK",
+        "body": {
+            "aad": {
+                "clientId": "e87c98aa-6f4b-4a72-95d0-fc830a64ee06",
+                "authority": "login.microsoftonline.com"
+            }
+        }
+    },
+    "/auth-servers.json": {
```

Comment:
Since it's just accessibility testing, we could reduce this data:
1. We can leave here just 1 fake OAuth server, and replace all there URLs and client IDs with some random data like `000-000-000`.
2. Same for APIs, just a couple of them should be enough, i.e. Echo API and a GraphQL API (@javierbo1989 can you please give Cristina a very simple one with just a few types in it, not SpaceX :)).
3. All these headers below are not necessary, we can leave just empty array.

Reply:
Good points, I will make these changes to reduce the data :)
Regarding the headers, some responses require the "content-type" header to know how to de-serialize the response, but we can remove the rest of the headers indeed.

Reply:
Also, regarding the default.json file I have made changes there - there were some accessibility issues with color contrast that were fixed inside the default.json. But I have made it compact back.